packages/create-styles/src/create-style-system/utils.js

```diff
@@ -167,3 +168,14 @@ export function compileInterpolatedStyles(interpolatedStyles, props) {
 
 	return compiledStyles;
 }
+
+/**
+ * Creates an interpolated className. This is used when styled components are
+ * interpolated in the template literal.
+ *
+ * @param {string} displayName The displayName of the component.
+ * @returns {string} The interpolated className.
+ */
+export function getInterpolatedClassName(displayName) {
+	return typeof displayName === 'string' ? `ic-${hash(displayName)}` : '';
```

Comment:
Should we raise a development-env warning that the displayName was undefined?

Reply:
Good idea. But maybe not here. Perhaps in `contextConnect()` or `useContext()`?

What you think :)

Reply:
Yup, that makes sense to me!

Reply:
I just pushed that update :). Lemme know if that works